ivy/data_classes/container/experimental/statistical.py

```diff
@@ -676,6 +676,7 @@ def static_nanmedian(
         *,
         axis: Optional[Union[Tuple[int], int]] = None,
         keepdims: bool = False,
+        overwrite_input: Optional[bool] = False,
```

Comment:
the type hints should be bool as only `True` or `False` are allowed as values

Reply:
@juliagsy  Need slight clarification on this - replacing the line with following work?
`overwrite_input: bool = False,`

Reply:
yep, that's right! Optional[..] is not needed (: